vsts/pipelines/_securityChecks.yml

```diff
@@ -10,6 +10,12 @@ steps:
     projects: '$(Build.SourcesDirectory)\src\**\*.csproj'
     arguments: '-o $(Build.BinariesDirectory)'
     zipAfterPublish: false
+    modifyOutputPath: false
+
+- task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
```

Comment:
why are we duplicating this task here? It is already part of _buildTemplates.yaml.

Reply:
I removed it, i think during rebase it came back, I will remove it again